app/assets/javascripts/govuk_publishing_components/analytics-ga4/gtm-page-views.js

```diff
@@ -13,6 +13,13 @@
             referrer: this.getReferrer(),
             title: this.getTitle(),
             status_code: this.getStatusCode()
+          },
+          publishing: {
+            document_type: this.getMetaContent('format'),
+            publishing_app: this.getMetaContent('publishing-application'),
+            rendering_app: this.getMetaContent('rendering-application'),
+            schema_name: this.getMetaContent('schema-name'),
+            content_id: this.getMetaContent('content-id')
           }
```

Comment:
Just curious, do we need this data in the GA4 event or is this mimicking what UA currently does? I can see a case for `content-id` and maybe `format`, but the rest feel unnecessary unless they are actively being use for filtering in reports etc.  If we go with the generic structure this feels like it belongs in `state`.

Reply:
Apparently there's a whole load of data we don't currently capture that is desirable, which is reflected in this new structure. Yeah, I agree it's a lot.